tests/boot/boot_to_desktop.pm

```diff
@@ -48,7 +49,7 @@ sub run {
         wait_serial('Welcome to SUSE Linux', $timeout) || die "System did not boot in $timeout seconds.";
     }
     else {
-        $self->wait_boot(bootloader_time => $timeout, nologin => $nologin);
+        $self->wait_boot(bootloader_time => $timeout, nologin => $nologin, ready_time => $ready_time);
```

Comment:
Same story, https://openqa.suse.de/tests/8574262#step/await_install/98 let's wait a bit longer

Reply:
C'mon, this test is not even related. I don't mind using more time at all, but now it's nearly doubled.
btw many of previous failures happened before timeout increase.

Reply:
@dzedro I imagined you standing next to Biden:

![](https://giphy.com/gifs/joe-biden-town-hall-SWoRKslHVtqEasqYCJ)

Reply:
I do change, run tests if something pop-up like this timeouts I add another change and run tests multiple times again.
Like last page tests should be relevant.